### ISSUE TITLE
pods->import() no longer processes array values

### ISSUE THREAD

**Issue by Budapresst** [pods-framework/pods#5284]
**Describe the bug**
If the data array has a field/key with an array value (e.g. multiselect picklist), then the item will import and a new item will be added, but the field/key with the array value will not be updated.

**To Reproduce**
1. Create a Pod with a multiselect simple relationship list.
2. create a data array (e.g.  $data = array('yourfield'=>array('val1','val2)) with an array value for your multiselect data field
3. add a new pod item with $pod->add($data);  The new pod item will add correctly and the multiselect field will be updated with the 2 values.
4. repeat the process, but use $pod->import($data); A new item will be created, but the multiselect field values will NOT be set.

**Expected behavior**
The import() function should support array values as documented here: https://pods.io/docs/code/pods-api/import/

**Screenshots**

## Pods Version
Please provide the version of Pods you are running: 2.7.12

## WordPress Environment

<details>
```
WordPress Version: 5.0.3

PHP Version: 7.2.14

MySQL Version: 5.5.5

Server Software: Apache

Your User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.98 Safari/537.36

Session Save Path: /var/cpanel/php/sessions/ea-php72

Session Save Path Exists: Yes

Session Save Path Writeable: Yes

Session Max Lifetime: 1440

Opcode Cache:

Apc: No
Memcached: No
OPcache: Yes
Redis: No
Object Cache:

APC: No
APCu: No
Memcache: No
Memcached: No
Redis: No
WPDB Prefix: do_

WP Multisite Mode: No

WP Memory Limit: 256M

Current Memory Usage: 18.216M

Current Memory Usage (real): 22.000M

Pods Network-Wide Activated: No

Pods Install Location: /home/doverhu/public_html/bosch/wp-content/plugins/pods/

Pods Tableless Mode Activated: No

Pods Light Mode Activated: No

Currently Active Theme: DoverBosch

Currently Active Plugins:

Brute Force Login Protection: 1.5.3
Disable Emojis: 1.2.0
Disable Post Via Email: 1.0.0
Disable Search: 1.1.0
Disable XML-RPC: 1.1.1
Duplicate Post: 3.2.2
EWWW Image Optimizer: 4.6.1
Header Cleanup: 1.2.0
Password Protected: 2.2.3
Pods - Custom Content Types and Fields: 2.7.12
Polylang: 2.5.1
Profile Builder: 2.9.5
Remove Dashboard Access: 1.1.3
Search by ID: 1.3
WordPress Importer: 0.6.4
WP-Sweep: 1.1.0
```
</details>

## Pods Package Export (helpful!)

```
Copy and Paste the JSON Export from **Pods Admin, Migrate: Packages, Export** in WordPress admin here
```

**Additional context**

**Possible Workaround**
If you have discovered a workaround, please include it below.

**Comment by quasel** [comment]
did it work for you with earlier versions?

**Comment by jamesgol** [comment]
This did work and was broken by this commit https://github.com/pods-framework/pods/commit/7c09cec37d49e455668ed859c06b6260856665d5

**Comment by JoryHogeveen** [comment]
@jamesgol Are you sure that is the first commit where it broke? Seems odd since this commit changes the code for strings, not arrays.

**Comment by jamesgol** [comment]
Yeah, the import code converts the arrays to a comma separated string.  When I was first trying to dig into it I stopped that and it actually worked but I don't know if that would break something else.

**Comment by JoryHogeveen** [comment]
Where does it convert arrays to strings? The commit you mentioned does the other way around.

**Comment by jamesgol** [comment]
https://github.com/pods-framework/pods/blob/2.x/classes/PodsAPI.php#L9000

**Comment by JoryHogeveen** [comment]
Check, @sc0ttkclark could you revisit your changes?
See https://github.com/pods-framework/pods/issues/2042
PR: https://github.com/pods-framework/pods/pull/5174
